examples/tools/scene_viewer/camera_controller_plugin.rs

```diff
@@ -158,16 +161,22 @@ fn camera_controller(
         // Handle mouse input
         let mut mouse_delta = Vec2::ZERO;
         if mouse_button_input.pressed(options.mouse_key_enable_mouse) || *move_toggled {
+            window.set_cursor_grab_mode(CursorGrabMode::Locked);
+            window.set_cursor_visibility(false);
+
             for mouse_event in mouse_events.iter() {
                 mouse_delta += mouse_event.delta;
             }
+        } else {
+            window.set_cursor_grab_mode(CursorGrabMode::None);
+            window.set_cursor_visibility(true);
         }
 
         if mouse_delta != Vec2::ZERO {
             // Apply look update
-            options.pitch = (options.pitch - mouse_delta.y * 0.5 * options.sensitivity * dt)
+            options.pitch = (options.pitch - mouse_delta.y / 180.0 * options.sensitivity)
```

Comment:
Can you pull this magic number out into a const so readers have a better sense of why it exists?

180 is particularly suspicious too :p

Reply:
Ya, I think the 180.0 just comes from a misunderstanding in Valorant's dev team and then just deciding to stick with it, because it ends up being used in radians, so it ends up being 1 dot from the mouse is PI/180 radian movement. But this is just kinda speculation.